### ISSUE TITLE
how to run the code

### ISSUE THREAD

**Issue by Gchinanty** [alterzero/DBPN-caffe#3]
I have installed matcaffe. And I have downloaded pretrained models.
But after I run DBPN_main_rgb.m at the root of DBPN-caffe which I have directly downloaded from GitHub. I only get the following results:
Bi cubic: NaN , NaN , NaN , NaN
dbpn: NaN , NaN , NaN , NaN
What's the matter with what l did? And could you tell me how I can get the correct results?
Thanks!!!

**Comment by alterzero** [comment]
@Gchinanty which test set did you use? is it jpg file?

**Comment by ynieWXJB** [comment]
I used png file as input. But I got the same results with Gchinanty.

**Comment by Gchinanty** [comment]
I used Set5 and it is *.png file just as your code showed. So I am very puzzled.
And now I have another question: what does 'train_4x_div_128_rgb.txt' mean in the 'DBPN_net_4x.prototxt'? Does it mean the input size? If not, could you tell me what is the input size of the net_4x?
Thanks a lot!

**Comment by Gchinanty** [comment]
I used Set5 and it is *.png file just as your code showed. So I am very puzzled.
And now I have another question: what does 'train_4x_div_128_rgb.txt' mean in the 'DBPN_net_4x.prototxt'? Does it mean the input size? If not, could you tell me what is the input size of the net_4x?
Thanks a lot! @alterzero

**Comment by alterzero** [comment]
I'm not sure what's the problem here. 
Did you use 'DBPN_net_4x.prototxt' for the evaluation? not 'DBPN_mat_4x.prototxt'?
Can you run caffe.set_mode_cpu()? or caffe.set_mode_gpu()?

'train_4x_div_128_rgb.txt' is used for training step which contains the list of your hdf5 files, the LR image size is 32x32.
